dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/SynchronizationController.java

```diff
@@ -68,29 +64,21 @@
     @Autowired
     private RestTemplate restTemplate;
 
-    @Autowired
-    private RenderService renderService;
-
     @PreAuthorize( "hasRole('ALL') or hasRole('F_EXPORT_DATA')" )
-    @PostMapping( "/dataPush" )
-    public void execute( HttpServletResponse response )
+    @PostMapping( value = "/dataPush", produces = APPLICATION_JSON_VALUE )
+    @ResponseBody
+    public ImportConflicts execute()
```

Comment:
FYI: This method could not be tested with spring MVC tests as it would not return data in test setup. Please look extra careful here.